source/content/guides/drupal-9-commerce.md

```diff
@@ -45,7 +45,7 @@ This guide covers installing [Drupal Commerce](https://drupalcommerce.org/), an
 1. Use the Terminus Build Tools plugin to create a new Drupal 9 site from the Pantheon [Drupal Recommended](https://github.com/pantheon-systems/drupal-composer-managed) repository on GitHub:
 
  ```bash{promptUser: user}
- terminus build:project:create pantheon-systems/d9 $SITENAME
+ terminus build:project:create d9 $SITENAME
```

Comment:
Pair this with a change to build tools similar to https://github.com/pantheon-systems/terminus-build-tools-plugin/commit/cc8a35cb9313bc21e0bc906c69862fadc6be7885